tasks/managed/update-trusted-tasks/README.md

```diff
@@ -10,3 +10,7 @@ If it is already in place, it will be used as an input to which the results will
 | Name       | Description                                                             | Optional | Default value |
 |------------|-------------------------------------------------------------------------|----------|---------------|
 | snapshotPath   | Path to the JSON string of the Snapshot spec in the data workspace | No       | -             |
+
+## Changes in 0.2.0
+
+Task's image (quay.io/konflux-ci/appstudio-utils) has been updated to the latest
```

Comment:
```suggestion
* Task's image (quay.io/konflux-ci/appstudio-utils) has been updated to the latest
```
It's a detail, but we use this format in (I hope) all the other files.

Reply:
Done.

The latest PRs were merged to development branch.
Is there a process to bump the development branch to production?

Reply:
> Is there a process to bump the development branch to production?

Yes, it happens every Wednesday (so will happen later today - usually around morning eastern time).